main.go

```diff
@@ -184,91 +169,92 @@ func (l *CompliancePlugin) PrepareForEval(req *proto.PrepareForEvalRequest) (*pr
 	// Local SSH Plugin: Fetch the SSH configuration from the local machine
 	// SAST Report Plugin: Convert a SAST sarif report into a usable structure for policies to be written against
 	// Azure VM Label Plugin: Collect all the VMs from the Azure API so they can be evaluated against policies
-	l.data = map[string]interface{}{
-		"foo": "bar",
-	}
+	l.data = knownVulnerabilities
 	return &proto.PrepareForEvalResponse{}, nil
 }
 
 func (l *CompliancePlugin) Eval(request *proto.EvalRequest) (*proto.EvalResponse, error) {
-
-	// Eval is used to run policies against the data you've collected in PrepareForEval.
-	// Eval will be called N times for every scheduled plugin execution where N is the amount of matching policies
-	// passed to the agent.
-
-	// When a user passes multiple policy bundles to the agent, each will be passed to Eval in turn to run against the
-	// same data collected in PrepareForEval.
-
 	ctx := context.TODO()
 	start_time := time.Now().Format(time.RFC3339)
 
-	// The Policy Manager aggregates much of the policy execution and output structuring.
-	results, err := policyManager.
-		New(ctx, l.logger, request.BundlePath).
-		Execute(ctx, "local_ssh", l.data)
+	response := runner.NewCallableEvalResponse()
+	hasViolations := false
+
+	for _, violation := range l.data {
+		violationMap := map[string]interface{}{
+			"cve_id":      violation.CVEID,
+			"title":       violation.Title,
+			"description": violation.Description,
+			"severity":    violation.Severity,
+			"remarks":     violation.Remarks,
+		}
 
-	if err != nil {
-		return &proto.EvalResponse{}, err
-	}
+		dataMap := map[string]interface{}{
+			"violation": []interface{}{violationMap},
+		}
```

Comment:
For the internal data map, I would replace violation with vulnerability for more clarity. 
In it's current form, we will evaluate `input.violation` in the policies, which causes some ambiguity between what is policy and what is data. 
I would suggest replacing the data root with `vulnerability` to bring more clarity to the policies. violation == "breaking policy" <-- judgement, vulnerability == "exists in the OS" <-- fact